integration-tests/docker-compose.yml

```
@@ -1,7 +1,7 @@
 version: '3'
 services:
   geth0:
-    image: "ethereum/client-go:v1.10.18"
+    image: "ethereum/client-go:v1.10.19"
```

```suggestion
    image: "ethereum/client-go:v1.11.15"
```
This should be the new release of go-ethereum https://hub.docker.com/layers/ethereum/client-go/v1.11.5/images/sha256-31822fb33f7db2a7155c0148f6f0e00fd9220bc9b00f0543895766b510461002?context=explore

---

Haha, you're right, it was late and confused it with the go 18->19 update (both were ending in 18). Fixed.

---

no worries :)